### PR TITLE
feat: adjust title of ahjo application record

### PR DESCRIPTION
**Pull request by rikuke**
## Description :sparkles:
Modify the Ahjo record title so, that if the record is for an application and it is a update request, the word "täydennys" is added to the record title.

### CODE REVIEW

File: backend/benefit/applications/services/ahjo_payload.py [hunk 1]
```diff
@@ -15,17 +20,23 @@
...
 ) -> str:
     """Prepare the title for the application record in Ahjo in the format:
     Hakemus 11.4.2024, 128123
     or for an attachment:
     Hakemus 11.4.2024, liite 1/3, 128123
+    If the request type is an update, add the word "täydennys" to the title.
     """
     formatted_date = application.created_at.strftime("%d.%m.%Y")
+    is_update = (
+        " täydennys," if request_type == AhjoRequestType.UPDATE_APPLICATION else ""
```
Comment (sirtawast) [inline backend/benefit/applications/services/ahjo_payload.py:35]: Using hardcoded static value is a bit sketchy.

**Review by sirtawast — CHANGES_REQUESTED** [review]

File: backend/benefit/applications/services/ahjo_payload.py (hunk 1 above)
Comment (sirtawast) [reply backend/benefit/applications/services/ahjo_payload.py:35]: Put it in a var at least.

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr3030.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr3030.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr3030.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe handler result is __failed__ for https://helsinkilisa-ui-handler-pr3030.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr3030.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr3030.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr3030.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr3030.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**